### PR TITLE
Feature/heroku

### PR DESCRIPTION
**Pull request by travissleavitt**
https://southern-exclamation.herokuapp.com

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "semi": false,
```
Comment (travissleavitt) [inline .prettierrc:2]: Can't believe I did this.

File: index.js [hunk 2]
```diff
@@ -76,36 +93,35 @@ const run = (req, rating = 'general') => {
...
-app.get('/', (req, res) => res.send(run(req, 'general')))
-app.post('/', (req, res) => res.send(run(req, 'general')))
-
-app.get('/all', (req, res) => res.send(run(req, 'all')))
-app.post('/all', (req, res) => res.send(run(req, 'all')))
-
-app.get('/adult', (req, res) => res.send(run(req, 'adult')))
-app.post('/adult', (req, res) => res.send(run(req, 'adult')))
-
-exports.getPossibleTargets = getPossibleTargets
```
Comment (travissleavitt) [inline index.js:108]: Stop testing API's in the browser. Use Postman or Insomnia.

File: .prettierrc (hunk 1 above)
Comment (ewendlick) [reply .prettierrc:2]: I truly appreciate you putting this rule in

Comment (travissleavitt) [reply .prettierrc:2]: I lost a little bit of my soul.

**Pull request merged**